docs/converters/index.rst

```diff
@@ -0,0 +1,36 @@
+.. _converters:
+
+Converters
+===============
+
+OAK uses a number of different Data Models (See  :ref:`datamodels`) for representing ontologies and
+ontology-related information.
+
+Command Line Usage
+-------------------
+
+The general pattern is:
+
+.. code-block:: bash
+
+    $ runoak -i INPUT-SPEC dump -O FORMAT -o OUTPUT-FILE
+
+Converting from Obo Graphs
+---------------------------
+
+Currently the only converters implemented are for converting between
```

Comment:
Converter to obograph might be nice at some point, but not sure if there is a need.

Reply:
And certainly we'll want direct converters without needing the obograph intermediary.